### ISSUE TITLE
--watch not watching imported dependencies outside of root dir

### ISSUE THREAD

**Issue by spiderworm** [michaelwayman/node-sass-chokidar#51]
- project has src/ dir
- parallel to src/, project has deps in node_modules/
  - some deps include .scss files I want to watch for changes as well (linked dependencies)
- some .scss files in src/ dir `@import` other .scss files from node_modules/
- making changes to .scss files in src/ triggers rebuild... but changes to .scss files in node_modules/ does not. I have to restart the process to see those updates

Is there a way to watch the .sass files imported from node_modules/ as well so that changes to those files will trigger a rebuild? I am guessing this would be done by having the option to disable the cache of files imported from outside the watched dir?

**Comment by spiderworm** [comment]
I worked around this for now by created a script that force restarts the node-sass-chokidar process every second to force it to get it to keep rebuilding the node_modules/ .scss files, but this is a silly thing to do and I don't want to do it any longer than I have to.

**Comment by michaelwayman** [comment]
I am not able to reproduce this.
When I have something like this

package.json
```
...
  "scripts": {
    "watchcss": "node-sass-chokidar --include-path ./node_modules ./src -o ./src --watch",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
```

and

src/app.scss
```
@import "_custom_bootstrap.scss";
@import "bootstrap/scss/bootstrap.scss";

a {
	color: red;
}
```

and `npm run watchcss`, I get `=> changed: /Users/wayman/projects/test-node-chokidar/src/app.scss`

and then I go to `node_modules/ bootstrap/scss/` and start making changes to the `scss` files in there, I get `=> changed: /Users/wayman/projects/test-node-chokidar/src/app.scss` again and the the changes I made to bootstrap were reflected in `app.css`

So basically I am unable to reproduce. I am closing this issue until someone else can verify this, or you can upload a project demonstrating the alleged issue.
